radar/data_management/bigquery.yaml

```diff
@@ -0,0 +1,25 @@
+name: Bigquery
+logo: https://commons.wikimedia.org/wiki/File:Google-BigQuery-Logo.svg
+blip:
+  - date: 2020-04-16
+    ring: TRIAL
+description: |
+  [Bigquery](https://cloud.google.com/bigquery) Serverless, highly scalable,
+  and cost-effective cloud data warehouse designed for business agility.
+rationale: |
+  Analyze petabytes of data using ANSI SQL at blazing-fast speeds,
+  with zero operational overhead.
+  Run analytics at scale with 26%–34% lower three-year TCO than cloud data warehouse alternatives.
+  Democratize insights with a trusted, and more secure platform that scales with your needs.
+license:
+  commercial:
+    company: Google
+    description: |
+      Use of this software requires a license for Google Cloud Platform.
+tags:
+  - cloud
+  - gcp
+  - commerical
```

Comment:
Remove commercial tag. It is automatically added based on license.

Reply:
I have removed it now.